### DISCUSSION TITLE
Fresh Ventura installer stops halfway

### DISCUSSION THREAD

**Discussion by Pondake** [osy86/HaC-Mini#1069] [Q&A]
I'm not so sure what I should share so let's start with everything that I've done and tried;

- Downloaded Ventura from the app store
- Erased the USB (Mac os extended (Journaled) & GUID Scheme)
- Moved the installed with the commands
- Run HacMini.pkg (latest; 3.2.2)
- Reset my BIOS using defaults and applied the 3 changes
- (Verified it was running the latest bios)
- Ran Opencore through the Boot menu (Sometimes through Efi shell because it didn't boot)
- The Apple logo + loadbar goes about halfway and won't progress (Waited a full day)

What I've tried;
- Enabling Legacy Boot, disabling network boot options
- Retraced the installation steps and prepared the USB multiple times
- Removing the SATA HDD
- Removing my NVMe SSD (Guide recommends clean install, but Windows is still installed. I'd love to reset the SSD but I can't figure out how if I can't get to the installer)
- Enabling verbose; it seems to stop after "SMCSuperIO ssio @ detected device intel nuc embedded controller". If any messages prior to this event is needed, please ask
- Switched USB sticks
- Removed one of my RAM sticks (2x 16GB) (Perhaps it's too much? Idk at this point)

Is there something obvious that I'm missing / I'm not seeing? I noticed Secure Boot is required per Monterey 12.2+ but I read that set-up is only needed for OTA updates. I'll gladly follow it, but I just want a working installation first.

**Comment by osy** [comment]
What’s your SSD model and do you have multiple SSD installed?

**Reply by Pondake** [reply]
I have a Intel SSD 600P installed (SSDPEKKW512G7). In the Sata slot I have a Hitachi 500GB HDD (7k750-500). No other SSD's installed.
Removing either one of them (or both) yields the same results though.

Rerunning through every letter in the guide; I noticed "Patching OS X installer" is disabled on my USB. The guide suggests that in this case, I need to run HacMini.pkg after Mac OS is installed. So I figure that's not the issue, but worth mentioning. 

Downloading Montery now just to keep on trying. I'm out of ideas :)

**Reply by osy** [reply]
I’ve ran into the same issue when using an incompatible SSD: https://github.com/dortania/bugtracker/issues/192

**Reply by Pondake** [reply]
I don't understand. The installer also doesn't run when the SSD is not installed. 
Am I supposed to buy a new SSD or do I somehow need to disabled TRIM? 😕

**Reply by osy** [reply]
Maybe take a high res picture of the verbose boot where it gets stuck and I can see if there’s any obvious issues.

**Reply by Pondake** [reply]
Of course!

![IMG20230319003537](https://user-images.githubusercontent.com/4643209/226145606-f43dd2df-6b95-42e3-a7c2-0dd5c20e7867.jpg)

**Reply by osy** [reply]
Are you on NUC8i7HNK or NUC8i7HVK?

**Reply by Pondake** [reply]
> Are you on NUC8i7HNK or NUC8i7HVK?

Neither, NUC8i5BEH2.. I guess that's the problem then 🥲

**Reply by osy** [reply]
Ah yeah, from the first page:

> The goal of this guide is to create the perfect Mac mini alternative using the Intel NUC Hades Canyon (NUC8i7HNK/NUC8i7HVK) and macOS. This guide is not for setting up a Hackintosh for any other machine. For a general Hackintosh guide, the [Vanilla Hackintosh Guide](https://hackintosh.gitbook.io/-r-hackintosh-vanilla-desktop-guide/) is highly recommended and is the main source of inspiration for this guide.

**Reply by osy** [reply]
It's not going to work because the ACPI is coded specifically for Hades Canyon.

**Reply by Pondake** [reply]
Okay thanks for the help. I'll try to follow another guide 😄
